playground/solid/App.jsx

```diff
@@ -0,0 +1,29 @@
+/** @jsxImportSource solid-js */
+/* eslint-disable no-restricted-globals */
+import { Navigation, Pagination, Scrollbar, A11y } from 'swiper';
+import { Swiper, SwiperSlide } from 'swiper/react/swiper-react';
```

Comment:
shouldn't it import from a solid folder?

Reply:
Yes, I missed this one.